content/WebApiHost/Startup.cs

```diff
@@ -65,14 +65,21 @@ public void ConfigureServices(IServiceCollection services)
 
             // Configures the NybusHostedService so that Nybus is started when the web application is started
             services.AddHostedService<NybusHostedService>();
-//#endif
-//#if (AddWcfDiscovery)
+            //#endif
 
-            // Configures the WCF Discovery from the current configuration
-            services.AddDiscovery(Configuration);
-            services.AddSingleton<IDiscoveryService, NetTcpDiscoveryService>();
-//#endif
-//#if (AddNybusBridge || ConfigureAWS)
+            //#if (AddDiscoveryAdapter)
+            services.ConfigureServiceDiscovery(o =>
+            {
+                o.ConfigureDiscoveryAdapterBinding = binding =>
+                {
+                    binding.Security.Mode = SecurityMode.None;
+                };
+            });
+            services.AddServiceDiscoveryAdapter();
+            services.AddBindingCustomization(binding => binding.Security.Mode = SecurityMode.None);
+
```

Comment:
Here you also need `services.ConfigureServiceDiscovery(Configuration.GetSection("Discovery"));` this line that you added before. You can have multiple calls to ConfigureServiceDiscovery.

Reply:
Also I would include this line `//services.DiscoverServiceUsingAdapter<IDiscoverableWcfService>();` but I will comment this code, this is just an example how you should register discoverable wcf service, where IDiscoverableWcfService in theory is the wcf service reference that we need to use in this project. You can check here how we usually set up discoverable wcf service 
https://github.com/keghub/content/blob/master/Clients/Jobs/Trigger/src/Trigger.Host/Extensions/ServiceCollectionWcfDiscoveryAdapterExtensions.cs